### PR TITLE
Docs: update license restrictions

### PR DESCRIPTION
**Pull request by AgnesToulet**
**What this PR does / why we need it**:
Update license documetation as we are deprecating tiered licensing

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/enterprise/license/license-restrictions.md [hunk 1]
```diff
@@ -12,135 +12,41 @@ When you become a Grafana Enterprise customer, you receive a license that govern
 
 You either have:
 
-- tiered licensing, or
-- combined licensing
+- [combined licensing](#combined-licensing), or
+- [tiered licensing](#tiered-licensing-deprecated) (deprecated)
 
-To understand which license type you have, refer to [Determine the number of active users for each licensed role](#determine-the-number-of-active-users-for-each-licensed-role).
```
Comment (gguillotte-grafana) [inline docs/sources/enterprise/license/license-restrictions.md:18]: Why is this line removed? This seems more relevant with the deprecation, so users can determine whether they need to update their licensing.

File: docs/sources/enterprise/license/license-restrictions.md [hunk 2]
```diff
@@ -12,135 +12,41 @@ When you become a Grafana Enterprise customer, you receive a license that govern
...
-
-After you apply the token, Grafana Enterprise resets your license and updates the user counts on the **Utilization** panel.
-
-> If you are running Grafana Enterprise 8.2 or earlier, the license grants you the total number of licensed users _for each user type_.
-
-For example, if your current license includes 60 viewers and 40 editor/admins, the new license includes 100 viewers and 100 editor/admins. Grafana Enterprise 8.3 removes the distinction between viewers and editor/admins as shown on the **Utilization** panel.
-
-Before you upgrade to Grafana 8.3, ensure that the total number of active users in Grafana does not exceed the number of users in your combined license. If it does, then new users cannot sign in to Grafana 8.3 until the active user count returns below the licensed limit.
-
-## Determining your current license type
```
Comment (gguillotte-grafana) [inline docs/sources/enterprise/license/license-restrictions.md:130]: This section seems more relevant with the deprecation, so users can determine whether they need to update their licensing, and IMO should not be removed.

File: docs/sources/enterprise/license/license-restrictions.md [hunk 3]
```diff
@@ -12,135 +12,41 @@ When you become a Grafana Enterprise customer, you receive a license that govern
...
-- **Review and update organization permissions.** You can modify organization permissions to reduce the count of one type of user. For example, if you reach the limit of editor/admin users, assign viewer permissions to one (or more) editor/admin users who agree to read-only access to dashboards. The license count updates when a user signs back in to Grafana.
-- **Review and update dashboard permissions.** Because viewers can also count toward editor/admin usage, if they are granted access to edit specific dashboards, review your dashboard permissions and adjust accordingly. Remove editor or admin permissions where necessary.
-- **Delete users.** Review the number of users you have for each type of permission and remove inactive users. Removing users immediately frees up space for new users.
-
-> When you delete a user you also delete their preferences, dashboard and folder permissions, and references to that user from the dashboard versions they have updated.
-
-Users who are synced via SAML, Oauth, or LDAP will also lose these attributes.
+1. Review the utilization count on the **Utilization** panel.
 
-### Update user permissions for dashboards
```
Comment (gguillotte-grafana) [inline docs/sources/enterprise/license/license-restrictions.md:94]: Should there be a note pointing to RBAC, which provides the ability to change the access certain roles of users have to specific dashboards regardless of license type, to replace this functionality being deprecated by tiered licenses?

**Review by gguillotte-grafana — COMMENTED** [review]
I'm curious as to why we should remove the docs for determining the license type, since it seems more users should be doing that with the deprecation in order to determine whether they need to update their license.

**Comment by gguillotte-grafana** [comment]
This is milestoned to 9.0 and labelled for no backport, but contains content quite relevant to users of 8.3. I suggest backporting to 8.3.

File: docs/sources/enterprise/license/license-restrictions.md (hunk 1 above)
Comment (AgnesToulet) [reply docs/sources/enterprise/license/license-restrictions.md:18]: This will no longer be available in 9.0. I guess we could replace this with "contact your Grafana account team"? cc @mjseaman

File: docs/sources/enterprise/license/license-restrictions.md (hunk 2 above)
Comment (AgnesToulet) [reply docs/sources/enterprise/license/license-restrictions.md:130]: Yes you're right, it's still relevant for users coming from before 8.3 to 9.0, I put it back in the doc.

File: docs/sources/enterprise/license/license-restrictions.md (hunk 3 above)
Comment (AgnesToulet) [reply docs/sources/enterprise/license/license-restrictions.md:94]: We talked about permissions here before because it had a strong impact on the active user count that was used for  licensing. As it has no impact anymore on licensing, I don't think it's relevant to keep it here anymore.

**Comment by AgnesToulet** [comment]
This PR doesn't contain any new information for 8.3. I just moved the `Combined licensing` part above the `Tiered licensing` because it should be the most important part of the documentation from 9.0 (as tiered licensing got deprecated).

It's not really easy to read as I removed / moved a lot of things but the only new information I added is:
> As of Grafana Enterprise Version 9.0, tiered licensing will be automatically treated as combined licensing.
For example, your license might include 300 viewers and 50 editors or administrators. This will be handled as a combined license for 350 users.

**Comment by gguillotte-grafana** [comment]
I see - thanks, @AgnesToulet. 8.3 users don't need to know that the feature is deprecated in 9.0, that's where I was confused. If users on 9.0 still have tiered licenses and need to manage users until they change to combined licensing, and can't find the information in the 9.0 docs, the docs for 8.3 should still be live.

I'm not sure if users will have the same confusion, but perhaps support should be aware that these docs are leaving in 9.0, but the feature will still be present.

File: docs/sources/enterprise/license/license-restrictions.md (hunk 2 above)
Comment (gguillotte-grafana) [reply docs/sources/enterprise/license/license-restrictions.md:130]: I'm sorry, I should have been clearer. The "transition to combined licensing model" section still appears to be relevant. Otherwise, I'm not sure how 9.0 users with tiered licensing will be able to transition to combined licensing.

File: docs/sources/enterprise/license/license-restrictions.md (hunk 1 above)
Comment (mjseaman) [reply docs/sources/enterprise/license/license-restrictions.md:18]: Agree @AgnesToulet

**Review by mjseaman — APPROVED** [review]
Made some copy updates, LGTM! Thanks @AgnesToulet.

**Comment by AgnesToulet** [comment]
> If users on 9.0 still have tiered licenses and need to manage users until they change to combined licensing, and can't find the information in the 9.0 docs, the docs for 8.3 should still be live.

@gguillotte-grafana This won't happen as, from 9.0, we handle tiered licenses as combined licenses even if the customer didn't upgrade (e.g. if a customer has a tiered licenses with 20 admins and 50 viewers, it will be handled like he has a combined license with 70 users in 9.0).

Also, Mitch did some updates and added this for older versions: 
> If you are running a pre-9.0 version of Grafana Enterprise, please refer to the documentation for that version to learn more about license enforcement in your current version.
